### ISSUE TITLE
Produce a KG-Hub logo

### ISSUE THREAD

**Issue by caufieldjh** [Knowledge-Graph-Hub/knowledge-graph-hub-support#10]
This has come up more than a few times, but a logo to represent KG-Hub would be helpful to have.
I'd favor a high-contract, minimal vectorizable logo. It should look good in both color and B&W.
Will post a few initial sketches here.

**Comment by caufieldjh** [comment]
Something along these lines:
![OIG](https://github.com/Knowledge-Graph-Hub/knowledge-graph-hub-support/assets/7987729/25e35cfc-44a8-443e-808c-394b3dedf4b6)
Though that really just says "graph" and not "graph infrastructure" or "knowledge graph"

**Comment by caufieldjh** [comment]
Or something more hub-like:
![hubpic](https://github.com/Knowledge-Graph-Hub/knowledge-graph-hub-support/assets/7987729/583f117d-b503-4da5-8141-6e3c8e3b84e6)

**Comment by caufieldjh** [comment]
Another simple one:
![simplekg](https://github.com/Knowledge-Graph-Hub/knowledge-graph-hub-support/assets/7987729/3ce85fed-5e36-4f45-827e-560e2f538616)

**Comment by caufieldjh** [comment]
And another wagon wheel-like hub:
![tgwheel](https://github.com/Knowledge-Graph-Hub/knowledge-graph-hub-support/assets/7987729/fc0d7f78-d875-4a67-8450-458c63912b5d)

(ignore the generative AI sketchiness)

**Comment by matentzn** [comment]
So Far I like the simple one. I think a logo should be a few strokes nothing too complicated.. and I think cognitively the hub/spoke analogy is not immediately obvious to most people.

**Comment by caufieldjh** [comment]
I agree - a simple logo will translate better to different icon sizes, too.

**Comment by nlharris** [comment]
Of those, I think I like the first one best (don't like the third one). But as you note, it just looks like a graph, not like a hub of many graphs. I'd like to see something a bit more fractal (while still simple enough for a logo).
